seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-doris/src/main/java/org/apache/seatunnel/flink/sink/DorisStreamLoad.java

```diff
@@ -104,7 +104,7 @@ public void load(String value) {
     private LoadResponse loadBatch(String data) {
         String formatDate = new SimpleDateFormat("yyyyMMdd_HHmmss").format(new Date());
         String label = String.format("flink_sink_%s_%s", formatDate,
-                UUID.randomUUID().toString().replaceAll("-", ""));
+                UUID.randomUUID().toString().replace("-", ""));
```

Comment:
The UUID will generate String with 4 `-`, maybe use `replaceAll` is more reasonable?

Reply:
String::replace does exactly the same thing as String::replaceAll without the performance drawback of the regex. just personal habit, , but there is no difference here.

Reply:
i restored it 🤣

Reply:
Ok, I just test and find they can return the same result.